tests/cephfs/kernel_update.py

```diff
@@ -13,30 +13,47 @@ def run(ceph_cluster, **kw):
     Update the kernel of client nodes
     Reboot the client nodes
     Verify kernel is updated
+    RHEL-8 package installation order:
+    kernel-core
+    kernel-modules
+    kernel
+    RHEL-9 package installation order:
+    kernel-modules-core
+    kernel-core
+    kernel-modules
+    kernel
     """
     try:
-        fs_util = FsUtils(ceph_cluster)
         clients = ceph_cluster.get_ceph_objects("client")
-
+        test_data = kw.get("test_data")
+        log.info("Test data is " + str(test_data))
+        url = test_data.get("custom-config")[0]
```

Comment:
Hi Julius,
custom-config can be passed in key value pairs.
instead of relaying on the url https://s3.upshift.redhat.com/DH-PROD-CKI/internal as their might be change in url. 
can we pass custom config as `--custom-config pre https://download.eng.bos.redhat.com/brewroot/vol/rhel-9/packages/kernel/5.14.0/388.el9/x86_64/kernel-5.14.0-388.el9.x86_64.rpm`
you can access it as a dict.

Reply:
@AmarnatReddy so we pass it as `--custom-config pre{URL}` or `--custom-config pre:{URL}` ?

Reply:
it is resolved.